src/main/java/org/codehaus/mojo/buildhelper/RemoveLocalArtifactMojo.java

```diff
@@ -89,6 +97,9 @@ public void execute()
         if ( removeAll )
         {
             localArtifactDirectory = localArtifactDirectory.getParentFile();
+        } else {
+            File metadataFile = new File( localArtifactDirectory, "maven-metadata-local.xml" );
```

Comment:
The "local" should be probably configurable ... but it can be done also in some future PR if someone would need it.
https://maven.apache.org/ref/3.2.5/maven-repository-metadata/index.html